beacon-chain/p2p/service.go

```diff
@@ -227,9 +227,9 @@ func (s *Service) Start() {
 	async.RunEvery(s.ctx, refreshRate, s.RefreshENR)
 	async.RunEvery(s.ctx, 1*time.Minute, func() {
 		log.WithFields(logrus.Fields{
-			"inbound":     len(s.peers.InboundConnected()),
-			"outbound":    len(s.peers.OutboundConnected()),
-			"activePeers": len(s.peers.Active()),
+			"inbound":  len(s.peers.InboundConnected()),
+			"outbound": len(s.peers.OutboundConnected()),
+			"active":   len(s.peers.Active()),
```

Comment:
This may break some scripts, any reasons to change? Although I agree it's a change for the better.

Reply:
No reason appart redundancy and consistency.

The log is:
```
INFO p2p: Peer summary activePeers=10 inbound=0 outbound=9
```

We already know that `peers` are the items being `active` since the log itself if `Peer summary`.

Also to be consistent, we would have either:

```
INFO p2p: Peer summary activePeers=10 inboundPeers=0 outboundPeers=9
```

or

```
INFO p2p: Peer summary active =10 inbound=0 outbound=9
```

But ok for scripts break, I'll fix.